lib/cli.rs

```diff
@@ -73,6 +73,7 @@ pub use simple_args::help as simple_args_help;
 pub async fn put_deploy(
     maybe_rpc_id: &str,
     node_address: &str,
+    maybe_block_identifier: &str,
```

Comment:
As with ordering the args in the subcommands, I'd probably try and keep the rpc_id, node_address and verbosity parameters grouping unbroken.

Reply:
I gotta say, if the grouping of parameters is that important, making them into a struct would make sense.